PLOT/donations.yml

```diff
@@ -38,4 +38,16 @@
     tel: (408)753-7032
   ticker: PLOT
   timestamp: 1643500678
```

Comment:
Please correct the format of the donations.yml file based on the README.
For a real world example check this [link](https://github.com/CardanoMDP/Donations/blob/main/SPEC/donations.yml) .
make sure about the correct space and intentation. Seems everything upon the other atm.

Reply:
LGTM now